### PR TITLE
Use `pi-hole` core repo for GitHub stars/forks

### PR DESCRIPTION
**Pull request by dschaper**
Signed-off-by: Dan Schaper <dan.schaper@pi-hole.net>

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 site_name: 'Pi-hole documentation'
 site_url: 'https://docs.pi-hole.net/'
-repo_url: 'https://github.com/pi-hole/docs'
+repo_url: 'https://github.com/pi-hole/pi-hole'
```
Comment (XhmikosR) [inline mkdocs.yml]: This isn't right. It will break with the edit button.

Comment (dschaper) [reply mkdocs.yml]: Meaning?

Comment (dschaper) [reply mkdocs.yml]: Screenshot_2020-02-09 Overview of Pi-hole

This is what is there now, and that is the wrong stars/forks. That needs to be the stars/forks for the core project, 20.6k Stars, 1.4k Forks.

Comment (XhmikosR) [reply mkdocs.yml]: Is the edit URL correct for each page?

Comment (dschaper) [reply mkdocs.yml]: If it comes down to having an edit link or the right number of stars/forks then we have the right number of stars/forks and no edit link.

Comment (XhmikosR) [reply mkdocs.yml]: All I'm saying is that this patches introduces 404 links for the Edit URLs.

Comment (dschaper) [reply mkdocs.yml]: Well, then something changed and introduced that in the last few days/weeks. We used to have the stars/forks for the core repo in that top area.

Comment (XhmikosR) [reply mkdocs.yml]: I know what changed (I changed it).

I pushed a patch which should make the edit URL work again.

**Comment by dschaper** [comment]
@PromoFaux Can you check this over?

File: mkdocs.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+edit_uri: '../docs/blob/master/docs/'
```
Comment (XhmikosR) [inline mkdocs.yml]: Do note that I didn't use `/edit/` here so that it doesn't require a login to view the file.

**Pull request merged**